galley/pkg/crd/validation/config.go

```diff
@@ -360,8 +388,12 @@ func (whc *WebhookConfigController) reconcile(stopCh <-chan struct{}) {
 				whc.createOrUpdateWebhookConfig()
 			}
 		case <-webhookChangedCh:
-			// reconcile the desired configuration
-			whc.createOrUpdateWebhookConfig()
+			if !whc.webhookParameters.EnableValidation {
```

Comment:
This would be easier to read if the conditional was inverted, i.e.

```golang
if whc.webhookParameters.EnableValidation {
    whc.createOrUpdateWebhookConfig()
} else {
    whc.deleteWebhookConfig()
}
```